app/src/main/java/fr/free/nrw/commons/upload/UploadController.java

```diff
@@ -93,7 +93,7 @@ public void startUpload(final Contribution contribution, final ContributionUploa
             contribution.setDescription("");
         }
 
-        String license = prefs.getString(Prefs.DEFAULT_LICENSE, Prefs.Licenses.CC_BY_SA_3);
+        String license = prefs.getString(Prefs.DEFAULT_LICENSE, null);
```

Comment:
Nice catch! My PR was wrong suggesting that this line is anything to do with a spinner being able to have one option. Therefore I'll just revert these lines to the original implementation as it was correct before.